### PR TITLE
Parses URLs in content automatically into links

### PR DESCRIPTION
**Pull request by juho-jaakkola**
This runs the anypage content through ```output/longtext``` view which parses URLs automatically into links and also makes theming easier by wrapping the output inside .elgg-output div.

@mrclay Would this break any of the use cases for the ```anypage/content``` view that you added in d78d8d2351b44fbc625d09f8e2ee06260becbba0?

### CODE REVIEW

File: views/default/anypage/content.php [hunk 1]
```diff
@@ -13,4 +13,6 @@
 	return;
 }
 
-echo $page->description;
+echo elgg_view('output/longtext', array(
```
Comment (mrclay) [inline views/default/anypage/content.php]: Maybe add a `$vars['raw']` option to just output the description property? I can update my plugin as needed.

Comment (juho-jaakkola) [inline views/default/anypage/content.php]: Do you mean:
```php
if (elgg_extract($vars, 'raw')) {
    echo $page->description;
} else {
    echo elgg_view('output/longtext', array('...');
}
```
And your plugin would then call:
```php
echo elgg_view('anypage/content, array(
    'path' => $path,
    'raw' => true,
));
```

Comment (juho-jaakkola) [inline views/default/anypage/content.php]: Out of curiosity, what is your use case for the raw content?

Comment (mrclay) [inline views/default/anypage/content.php]: I don't remember how I'm using it. We probably don't need raw access. LGTM

**Pull request merged**